Microsoft.Toolkit.Uwp/Helpers/DispatcherHelper.cs

```diff
@@ -65,7 +65,12 @@ public static Task ExecuteOnUIThreadAsync(CoreApplicationView viewToExecuteOn, F
                 viewToExecuteOn,
                 async () =>
                 {
-                    await function().ConfigureAwait(false);
+                    var task = function();
+                    if (task == null)
+                    {
+                        throw new InvalidOperationException("The Task returned by function cannot be null.");
+                    }
+                    await task.ConfigureAwait(false);
```

Comment:
We have to await it because the signature of the lambda is different. 

function is type `Func<Task>` and the lambda using in that call is `Func<Task<object>>`

By the way, this call with lambda creation was found to be redundant and was removed in my other commit. @lukasf didn't know and went ahead to fix the other error that we found.